common/src/utils/clear_temp_files_dir.rs

```diff
@@ -0,0 +1,23 @@
+use std::{fs, io, path::PathBuf};
+
+use crate::STATIC_ARGS;
+
+pub fn clear_temp_files_directory(path_to_delete_specific_file: Option<PathBuf>) -> io::Result<()> {
+    match path_to_delete_specific_file {
```

Comment:
More of a nikpik but it might be best to keep variable fields short but still relevant to what its meant to be used for from the function. Eg renaming it to `path` should be enough. 

```suggestion
pub fn clear_temp_files_directory(path: Option<PathBuf>) -> io::Result<()> {
    match path {
```